res/values/strings_tl.arb

```diff
@@ -0,0 +1,681 @@
+  "send_new" : "Bago",
+  "send_amount" : "Halaga:",
+  "send_fee" : "Bayad:",
+  "send_name" : "Pangalan",
+  "send_got_it" : "Nakuha ko",
+  "send_sending" : "Nagpapadala...",
+  "send_success" : "Matagumpay na naipadala ang iyong ${crypto} ",
+
+
+  "settings_title" : "Mga Setting",
+  "settings_nodes" : "Mga Node",
+  "settings_current_node" : "Kasalukuyang node",
+  "settings_wallets" : " Mga Wallet",
+  "settings_display_balance" : "Balanse ng display",
+  "settings_currency" : "Pera",
+  "settings_fee_priority" : "Priyoridad sa bayad",
+  "settings_save_recipient_address" : "I-save ang address ng tatanggap",
+  "settings_personal" : "Personal",
+  "settings_change_pin" : "Palitan ang PIN",
+  "settings_change_language" : "baguhin ang wika",
+  "settings_allow_biometrical_authentication" : "Payagan ang biometrical authentication",
+  "settings_dark_mode" : "Madilim na mode
+  "settings_transactions" : "Mga Transaction",
+  "settings_trades" : "Mga kalakalan",
+  "settings_display_on_dashboard_list" : "Ipakita sa listahan ng dashboard",
+  "settings_all" : "LAHAT",
+  "settings_only_trades" : "Nangangalakal lamang",
+  "settings_only_transactions" : "Mga transaksyon lamang",
+  "settings_none" : "Wala",
+  "settings_support" : "Suporta",
+  "settings_terms_and_conditions" : "Mga tuntunin at kundisyon",
+  "pin_is_incorrect" : "Mali ang PIN",
+
+
+  "setup_pin" : "I-Setup ang PIN",
+  "enter_your_pin_again" : "Ilagay muli ang iyong PIN",
+  "setup_successful" : "Matagumpay na na-set up ang iyong PIN!",
+
+
+  "wallet_keys" : "Wallet seed/susi",
+  "wallet_seed" : "Wallet seed",
+  "private_key" : "Pribadong susi",
+  "public_key" : "Pampublikong susi",
+  "view_key_private" : "tingnan ang susi(pribado)",
+  "view_key_public" : "View key (pampubliko)",
+  "spend_key_private" : "Susi sa paggastos (pribado)",
+  "spend_key_public" : "Susi sa paggastos (pampubliko)",
+  "copied_key_to_clipboard" : "Kinopya ang ${key} sa Clipboard",
+
+
+  "new_subaddress_title" : "Bagong address",
+  "new_subaddress_label_name" : "Pangalan ng Label",
+  "new_subaddress_create" : "Lumikha",
+
+  "address_label" : "label ng address",
+
+  "subaddress_title" : "Listahan ng Subaddress",
+
+
+  "trade_details_title" : "Mga detalye ng kalakalan",
+  "trade_details_id" : "ID",
+  "trade_details_state" : "Katayuan",
+  "trade_details_fetching" : "Kumukuha",
+  "trade_details_provider" : "Provider",
+  "trade_details_created_at" : "nilikha sa",
+  "trade_details_pair" : "Pares",
+  "trade_details_copied" : "${title} Kinopya sa Clipboard",
+
+
+  "trade_history_title" : "Kasaysayan ng kalakalan",
+
+
+  "transaction_details_title" : "Mga detalye ng transakyson",
+  "transaction_details_transaction_id" : "ID ng transaksyon",
+  "transaction_details_date" : "Petsa",
+  "transaction_details_height" : "Taas",
+  "transaction_details_amount" : "Halaga",
+  "transaction_details_fee" : "Bayad",
+  "transaction_details_copied" : "${title} copied to Clipboard",
```

Comment:
Is is good to leave this fully English? In 284 you use *Kinopya* (which might be better lowercase as well ...)